modules/ROOT/pages/java.adoc

```diff
@@ -842,6 +842,154 @@ The level of compression is set on Sync Gateway and can be tuned in the configur
 To optimize for speed, the replication protocol doesn't guarantee that documents will be received in a particular order.
 So we don't recommend to rely on that when using the replication or database change listeners for example.
 
+==== Replicator Notifications on a Custom Executor
+
+Prior to version 2.6, Couchbase Lite spun up multiple executors.
+This policy could result in too many threads being spun up.
+
+An executor manages a pool of threads and, perhaps, a queue in front of the executor to handle the asynchronous callbacks.
```

Comment:
comma after "executor":  "a queue in front of the executor, to handle...".  It is the pool of threads *and* the queue that handle.

Reply:
```suggestion
An executor manages a pool of threads and, perhaps, a queue in front of the executor, to handle the asynchronous callbacks.
```